approved/v93k/flow_control.tf

```diff
@@ -1655,50 +1655,28 @@ test_flow
   }
   else
   {
-    @T1_BEA7F3B_FAILED = 1;
-  }
-  if @T1_BEA7F3B_FAILED == 1 then
-  {
     run_and_branch(erase1_BEA7F3B)
     then
     {
     }
     else
     {
-      stop_bin "", "fail", , bad, noreprobe, red, 12, over_on;
```

Comment:
Yes, thanks.  I think the (continue) that is originally with the set-flag is overriding the binning of substituted block from on-flag condition.  Will fix.